lib/data/external/GcpClient.js

```diff
@@ -28,17 +37,18 @@ class GcpClient extends AwsClient {
         this.type = 'GCP';
         this._gcpBucketName = config.bucketName;
         this._mpuBucketName = config.mpuBucket;
-        this._overflowBucketName = config.overflowBucket;
+        this._createGcpKey = this._createAwsKey.bind(this);
         this._gcpParams = Object.assign(this._s3Params, {
             mainBucket: this._gcpBucketName,
             mpuBucket: this._mpuBucketName,
         });
         this._client = new GCP(this._gcpParams);
+        this.listParts = undefined;
```

Comment:
Use underscore if this is a private property that is not expected to use outside this class.

Reply:
this is a reassignment to the inherited listParts method from AwsClient to use metadata listing instead of making a request to GCP. Will add comment to this.

Reply:
Ok.